core/templates/components/profile-link-directives/circular-image.component.ts

```diff
@@ -27,7 +28,14 @@ import { downgradeComponent } from '@angular/upgrade/static';
 export class CircularImageComponent {
   @Input() src: string;
   @Input() link?: string;
-  constructor() {}
+  constructor(
+    private sanitizer: DomSanitizer
+  ) {}
+
+  getSanitizedUrl(url: string): SafeUrl {
+    return this.sanitizer.bypassSecurityTrustUrl(url);
+  }
+
```

Comment:
Move this into a pipe. This will come in handy later on.

```

import { Pipe, PipeTransform } from '@angular/core';
import { DomSanitizer, SafeHtml, SafeStyle, SafeScript, SafeUrl, SafeResourceUrl } from '@angular/platform-browser';

@Pipe({
  name: 'safe'
})
export class SafePipe implements PipeTransform {

  constructor(protected sanitizer: DomSanitizer) {}
 
 public transform(value: any, type: string): SafeHtml | SafeStyle | SafeScript | SafeUrl | SafeResourceUrl {
    switch (type) {
			case 'html': return this.sanitizer.bypassSecurityTrustHtml(value);
			case 'style': return this.sanitizer.bypassSecurityTrustStyle(value);
			case 'script': return this.sanitizer.bypassSecurityTrustScript(value);
			case 'url': return this.sanitizer.bypassSecurityTrustUrl(value);
			case 'resourceUrl': return this.sanitizer.bypassSecurityTrustResourceUrl(value);
			default: throw new Error(`Invalid safe type specified: ${type}`);
		}
  }
}
```

Reply:
Okay, I'll do that

Reply:
Also, add a linter check to prevent its usage except for a file list that we specify.

Reply:
I think we shouldn't do that with this PR? I'll make a separate one once I figure out how to fix the linter checks

Reply:
@orthodoxparadox If you're doing that, please file an issue, assign it to yourself, and add the issue number to this comment thread.

In general, follow the above process for any deferred work, to ensure that nothing slips through the cracks. Thanks!

Reply:
Okay, adding an issue!

Reply:
@srijanreddy98 I have added the pipe and it's corresponding spec, ptal!